composer.json

```diff
@@ -14,7 +14,8 @@
         "symfony/framework-bundle" : "~3.0"
     },
     "require-dev" : {
-        "atoum/atoum" : "master-dev"
+        "atoum/atoum" : "master-dev",
+        "symfony/yaml": "~3.0"
```

Comment:
my bad, you're right, when composer install a dependency, it installs only the `require` section of the package... 
so the `symfony/yaml` package in `require-dev` of `symfony/framework-bundle` won't be installed

👍